src/components/CallView/shared/ReactionToaster.vue

```diff
@@ -28,7 +28,13 @@
 			:key="toast.seed"
 			class="toast"
 			:style="styled(toast.name, toast.seed)">
-			<span class="toast__reaction">
+			<img v-if="toast.reactionURL"
+				class="toast__reaction-img"
+				:src="toast.reactionURL"
+				:alt="toast.reaction"
+				width="32"
+				height="32">
```

Comment:
```suggestion
			<picture v-if="toast.reactionURL">
				<source :srcset="toast.reactionURL.replace('gif', 'webp')" type="image/webp">
				<img class="toast__reaction-img"
					:src="toast.reactionURL"
					:alt="toast.reaction"
					width="32"
					height="32">
			</picture>
```

That's a rough solution, but should do the trick, if we provide both .webp and .gif

Reply:
I will reduce the gif dimensions to 64px which leads to 91% for Concerned emoji for example. This makes gif have less size than webp version. 

p.s: I tried to resize the webp as it is in 512x512 too, but couldn't find a good resizing tool.